### PR TITLE
[docs-only] update env_vars.yaml post Activitylog API merge

### PR DESCRIPTION
**Pull request by mmattel**
References: #9361 (Activity API)

After #9361 was merged, we need to update the helper yaml file to show correct values in docs.

### CODE REVIEW

**Review by aduffeck — APPROVED** [review]

**Comment by mmattel** [comment]
CI hangs, no PR issue

File: docs/helpers/env_vars.yaml [hunk 1]
```diff
@@ -7256,131 +7440,132 @@ OCIS_ASYNC_UPLOADS:
...
   type: '[]string'
-  description: A list of nodes to access the configured store. Note that the behaviour
-    how nodes are used is dependent on the library of the configured store. See the
-    Environment Variable Types description for more details.
-  introductionVersion: "5.0"
+  description: A list of nodes to access the configured store. This has no effect
+    when 'memory' or 'ocmem' stores are configured. Note that the behaviour how nodes
+    are used is dependent on the library of the configured store. See the Environment
+    Variable Types description for more details.
+  introductionVersion: pre5.0
```
Comment (phil-davis) [inline docs/helpers/env_vars.yaml:7510]: These are "interesting", changing from "5.0" to "pre5.0"

File: docs/helpers/env_vars.yaml [hunk 2]
```diff
@@ -1,3 +1,149 @@
+ACTIVITYLOG_CORS_ALLOW_CREDENTIALS:
+  name: OCIS_CORS_ALLOW_CREDENTIALS;ACTIVITYLOG_CORS_ALLOW_CREDENTIALS
+  defaultValue: "true"
+  type: bool
+  description: 'Allow credentials for CORS.See following chapter for more details:
+    *Access-Control-Allow-Credentials* at https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials.'
+  introductionVersion: pre5.0
```
Comment (phil-davis) [inline docs/helpers/env_vars.yaml:7]: Maybe I do not understand the process.
How can these new lines have `introductionVersion: pre5.0` ?

**Review by phil-davis — COMMENTED** [review]
A couple of questions.

File: docs/helpers/env_vars.yaml (hunk 2 above)
Comment (mmattel) [reply docs/helpers/env_vars.yaml:7]: The process is the following:

`make -C docs docs-generate` parses all the envvars from all services and adds them to a kind of "database" file (this one) for further actions we need to generate the final .adoc files. The helper takes whatever devs have done with envvars in services, therefore the helper is "neutral" about the content. To distinguish when an envvar has been added, we needed to decide which one have been introduced before 5.0. Having that, we can now clearly identify when an envvar has been introduced, respctively will get deprecated.

With an upcoming version of the helper, we will add to the adoc output WHEN the respective envvar got introduced in the admin docs. Also, we can now provide easily a list of introduced envvars for a particular release 😄 

Hope this makes sense.

**Comment by micbar** [comment]
please rebase

**Comment by mmattel** [comment]
@micbar rebase done + commit with updates containing the latest changes 😃

**Pull request merged**